src/blocks/editPassword/editPassword.ts

```diff
@@ -0,0 +1,95 @@
+import renderDOM from '../../utils/render-dom';
+import PictureBlock from '../../components/picture-block';
+import Form from '../../components/form';
+import ProfileBlock from '../../components/profile-block';
+import ProfileItem from '../../components/profile-item';
+import Button from '../../components/button';
+import Validation from '../../utils/validation';
+import Event from '../../utils/event';
+import {INPUT_NAME} from '../../const';
+
+const pictureBlock = new PictureBlock({});
+
+const formWrapper = new ProfileBlock({
+  id: 'form'
+});
+
+const oldPasswordItem = new ProfileItem({
+  name: 'oldPassword',
+  type: 'password',
+  label: 'Старый пароль',
+  isInput: true,
+  events: {
+    'focusout': (e: Event) => Event.onInputFocusOut(e, Validation.checkPassword, oldPasswordItem)
+  }
```

Comment:
Лучше ставить в последних строках объектов в столбик `,` - это положительно сказывается на диффах в системах контроля версий (1 диф заместо 3)
```suggestion
  name: 'oldPassword',
  type: 'password',
  label: 'Старый пароль',
  isInput: true,
  events: {
    'focusout': (e: Event) => Event.onInputFocusOut(e, Validation.checkPassword, oldPasswordItem)
  },
```